### ISSUE TITLE
System.ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by ohjima** [PowerShell/PSReadLine#1069]
Environment data
----------------

PS C:\Dev\Python\workspace\lf> cd 'c:\Dev\Python\workspace\lf'; ${env:ALLUSERSPROFILE}='C:\ProgramData'; ${env:AMD_ENTRYPOINT}='vs/workbench/services/extensions/node/extensionHostProcess'; ${env:ANDROID_HOME}='C:\Dev\Android'; ${env:APPDATA}='C:\Users\woo\AppData\Roaming'; ${env:APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL}='true'; 
${env:CommonProgramFiles}='C:\Program Files\Common Files'; ${env:CommonProgramFiles(x86)}='C:\Program Files (x86)\Common Files'; ${env:CommonProgramW6432}='C:\Program Files\Common Files'; ${env:COMPUTERNAME}='DESKTOP-HU1TADK'; ${env:ComSpec}='C:\WINDOWS\system32\cmd.exe'; ${env:DriverData}='C:\Windows\System32\Drivers\DriverData'; ${env:ELECTRON_RUN_AS_NODE}='1'; ${env:FPS_BROWSER_APP_PROFILE_STRING}='Internet Explorer'; ${env:FPS_BROWSER_USER_PROFILE_STRING}='Default'; ${env:HOMEDRIVE}='C:'; 
${env:HOMEPATH}='\Users\woo'; ${env:IntelliJ IDEA}='C:\Dev\Java\IntelliJ IDEA 2019.1\bin;'; ${env:JAVA_HOME}='C:\Dev\Java\jdk1.8.0_201'; ${env:LOCALAPPDATA}='C:\Users\woo\AppData\Local'; ${env:LOGONSERVER}='\\DESKTOP-HU1TADK'; ${env:NUMBER_OF_PROCESSORS}='12'; ${env:OS}='Windows_NT'; ${env:PATHEXT}='.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW'; ${env:PIPE_LOGGING}='true'; ${env:PROCESSOR_ARCHITECTURE}='AMD64'; ${env:PROCESSOR_IDENTIFIER}='Intel64 Family 6 Model 158 Stepping 
10, GenuineIntel'; ${env:PROCESSOR_LEVEL}='6'; ${env:PROCESSOR_REVISION}='9e0a'; ${env:ProgramData}='C:\ProgramData'; ${env:ProgramFiles}='C:\Program Files'; ${env:ProgramFiles(x86)}='C:\Program Files (x86)'; ${env:ProgramW6432}='C:\Program Files'; ${env:PSModulePath}='C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules'; ${env:PUBLIC}='C:\Users\Public'; ${env:SESSIONNAME}='Console'; ${env:SystemDrive}='C:'; ${env:SystemRoot}='C:\WINDOWS'; ${env:TEMP}='C:\Users\woo\AppData\Local\Temp'; ${env:TMP}='C:\Users\woo\AppData\Local\Temp'; ${env:USERDOMAIN}='DESKTOP-HU1TADK'; ${env:USERDOMAIN_ROAMINGPROFILE}='DESKTOP-HU1TADK'; ${env:USERNAME}='woo'; ${env:USERPROFILE}='C:\Users\woo'; ${env:VBOX_MSI_INSTALL_PATH}='C:\Program Files\Oracle\VirtualBox\'; ${env:VERBOSE_LOGGING}='true'; ${env:VSCODE_CWD}='C:\Users\woo\AppData\Local\Programs\Microsoft VS Code'; ${env:VSCODE_HANDLES_UNCAUGHT_ERRORS}='true'; ${env:VSCODE_IPC_HOOK}='\\.\pipe\bf540a6f1434f810277c8e9e397d1e34-1.38.1-main-sock'; ${env:VSCODE_IPC_HOOK_EXTHOST}='\\.\pipe\vscode-ipc-f961af96-c2de-4ecd-be36-cdc2e92ab0ae-sock'; ${env:VSCODE_LOGS}='C:\Users\woo\AppData\Roaming\Code\logs\20190925T161650'; ${env:VSCODE_LOG_STACK}='false'; ${env:VSCODE_NLS_CONFIG}='{"locale":"ko","availableLanguages":{},"_languagePackSupport":true}'; ${env:VSCODE_NODE_CACHED_DATA_DIR}='C:\Users\woo\AppData\Roaming\Code\CachedData\b37e54c98e1a74ba89e03073e5a3761284e3ffb0'; ${env:VSCODE_PID}='11732'; ${env:VSCODE_PREVENT_FOREIGN_INSPECT}='true'; ${env:windir}='C:\WINDOWS'; ${env:Path}='C:\Dev\Python\Python37\Scripts\;C:\Dev\Python\Python37\;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files (x86)\dotnet\;C:\Program Files\PuTTY\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client 
SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Dev\Java\IntelliJ IDEA 2019.1\bin;C:\Dev\Java\jdk1.8.0_201\bin;C:\Program Files\Tesseract-OCR;C:\Users\woo\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files (x86)\Microsoft\ILMerge;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Users\woo\AppData\Roaming\npm;C:\Program Files\nodejs;C:\Dev\Android\Flutter\bin;'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Dev\Python\Python37\python.exe' 'c:\Users\woo\.vscode\extensions\ms-python.python-2019.9.34911\pythonFiles\ptvsd_launcher.py' '--default' '--nodebug' '--client' '--host' 'localhost' '--port' '58517' 'c:\Dev\Python\workspace\lf\L
문제가 발생했습니다. 아래 세부 정보와 함께 이 버그를 보고해 주세요.
GitHub에 대한 보고: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
마지막 200 키:
 e r s \ w o o \ . v s c o d e \ e x t e n s i o n s \ m s - p y t h o n . p y t h o n - 2 0 1 9 . 9 . 3 4 9 1 1 \ p y t h o n F i l e s \ p t v s d _ l a u n c h e r . p y ' Space ' - - d e f a u l t ' Space ' - - n o d e b u g ' Space ' - - c l i e n t ' Space ' - - h o s t ' Space ' l o c a l h o s t ' Space ' - - p o r t ' Space ' 5 8 5 1 7 ' Space ' c : \ D e v \ P y t h o n \ w o r k s p a c e \ l f \ L F R m s . p y ' Space Enter

예외:
System.ArgumentOutOfRangeException: 값은 0보다 크거나 같아야 하며 해당 범위에서 콘솔의 버퍼 크기보다 작아야 합니다.
매개 변수 이름: top
실제 값은 -4입니다.
   위치: System.Console.SetCursorPosition(Int32 left, Int32 top)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   위치: Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   위치: Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Dev\Python\workspace\lf> cd 'c:\Dev\Python\workspace\lf'; ${env:ALLUSERSPROFILE}='C:\ProgramData'; ${env:AMD_ENTRYPOINT}='vs/workbench/services/extensions/node/extensionHostProcess'; ${env:ANDROID_HOME}='C:\Dev\Android'; ${env:APPDATA}='C:\Users\woo\AppData\Roaming'; ${env:APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL}='true'; 
${env:CommonProgramFiles}='C:\Program Files\Common Files'; ${env:CommonProgramFiles(x86)}='C:\Program Files (x86)\Common Files'; ${env:CommonProgramW6432}='C:\Program Files\Common Files'; ${env:COMPUTERNAME}='DESKTOP-HU1TADK'; ${env:ComSpec}='C:\WINDOWS\system32\cmd.exe'; ${env:DriverData}='C:\Windows\System32\Drivers\DriverData'; ${env:ELECTRON_RUN_AS_NODE}='1'; ${env:FPS_BROWSER_APP_PROFILE_STRING}='Internet Explorer'; ${env:FPS_BROWSER_USER_PROFILE_STRING}='Default'; ${env:HOMEDRIVE}='C:'; 
${env:HOMEPATH}='\Users\woo'; ${env:IntelliJ IDEA}='C:\Dev\Java\IntelliJ IDEA 2019.1\bin;'; ${env:JAVA_HOME}='C:\Dev\Java\jdk1.8.0_201'; ${env:LOCALAPPDATA}='C:\Users\woo\AppData\Local'; ${env:LOGONSERVER}='\\DESKTOP-HU1TADK'; ${env:NUMBER_OF_PROCESSORS}='12'; ${env:OS}='Windows_NT'; ${env:PATHEXT}='.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW'; ${env:PIPE_LOGGING}='true'; ${env:PROCESSOR_ARCHITECTURE}='AMD64'; ${env:PROCESSOR_IDENTIFIER}='Intel64 Family 6 Model 158 Stepping 
10, GenuineIntel'; ${env:PROCESSOR_LEVEL}='6'; ${env:PROCESSOR_REVISION}='9e0a'; ${env:ProgramData}='C:\ProgramData'; ${env:ProgramFiles}='C:\Program Files'; ${env:ProgramFiles(x86)}='C:\Program Files (x86)'; ${env:ProgramW6432}='C:\Program Files'; ${env:PSModulePath}='C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules'; ${env:PUBLIC}='C:\Users\Public'; ${env:SESSIONNAME}='Console'; ${env:SystemDrive}='C:'; ${env:SystemRoot}='C:\WINDOWS'; ${env:TEMP}='C:\Users\woo\AppData\Local\Temp'; ${env:TMP}='C:\Users\woo\AppData\Local\Temp'; ${env:USERDOMAIN}='DESKTOP-HU1TADK'; ${env:USERDOMAIN_ROAMINGPROFILE}='DESKTOP-HU1TADK'; ${env:USERNAME}='woo'; ${env:USERPROFILE}='C:\Users\woo'; ${env:VBOX_MSI_INSTALL_PATH}='C:\Program Files\Oracle\VirtualBox\'; ${env:VERBOSE_LOGGING}='true'; ${env:VSCODE_CWD}='C:\Users\woo\AppData\Local\Programs\Microsoft VS Code'; ${env:VSCODE_HANDLES_UNCAUGHT_ERRORS}='true'; ${env:VSCODE_IPC_HOOK}='\\.\pipe\bf540a6f1434f810277c8e9e397d1e34-1.38.1-main-sock'; ${env:VSCODE_IPC_HOOK_EXTHOST}='\\.\pipe\vscode-ipc-f961af96-c2de-4ecd-be36-cdc2e92ab0ae-sock'; ${env:VSCODE_LOGS}='C:\Users\woo\AppData\Roaming\Code\logs\20190925T161650'; ${env:VSCODE_LOG_STACK}='false'; ${env:VSCODE_NLS_CONFIG}='{"locale":"ko","availableLanguages":{},"_languagePackSupport":true}'; ${env:VSCODE_NODE_CACHED_DATA_DIR}='C:\Users\woo\AppData\Roaming\Code\CachedData\b37e54c98e1a74ba89e03073e5a3761284e3ffb0'; ${env:VSCODE_PID}='11732'; ${env:VSCODE_PREVENT_FOREIGN_INSPECT}='true'; ${env:windir}='C:\WINDOWS'; ${env:Path}='C:\Dev\Python\Python37\Scripts\;C:\Dev\Python\Python37\;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files (x86)\dotnet\;C:\Program Files\PuTTY\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client 
SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Dev\Java\IntelliJ IDEA 2019.1\bin;C:\Dev\Java\jdk1.8.0_201\bin;C:\Program Files\Tesseract-OCR;C:\Users\woo\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files (x86)\Microsoft\ILMerge;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Users\woo\AppData\Roaming\npm;C:\Program Files\nodejs;C:\Dev\Android\Flutter\bin;'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Dev\Python\Python37\python.exe' 'c:\Users\woo\.vscode\extensions\ms-python.python-2019.9.34911\pythonFiles\ptvsd_launcher.py' '--default' '--nodebug' '--client' '--host' 'localhost' '--port' '58517' 'c:\Dev\Python\workspace\lf\LF
문제가 발생했습니다. 아래 세부 정보와 함께 이 버그를 보고해 주세요.
GitHub에 대한 보고: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
마지막 200 키:
 e r s \ w o o \ . v s c o d e \ e x t e n s i o n s \ m s - p y t h o n . p y t h o n - 2 0 1 9 . 9 . 3 4 9 1 1 \ p y t h o n F i l e s \ p t v s d _ l a u n c h e r . p y ' Space ' - - d e f a u l t ' Space ' - - n o d e b u g ' Space ' - - c l i e n t ' Space ' - - h o s t ' Space ' l o c a l h o s t ' Space ' - - p o r t ' Space ' 5 8 5 1 7 ' Space ' c : \ D e v \ P y t h o n \ w o r k s p a c e \ l f \ L F R m s . p y ' Space Enter

예외:
System.ArgumentOutOfRangeException: 값은 0보다 크거나 같아야 하며 해당 범위에서 콘솔의 버퍼 크기보다 작아야 합니다.
매개 변수 이름: top
실제 값은 -4입니다.
   위치: System.Console.SetCursorPosition(Int32 left, Int32 top)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   위치: Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   위치: Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Dev\Python\workspace\lf> cd 'c:\Dev\Python\workspace\lf'; ${env:ALLUSERSPROFILE}='C:\ProgramData'; ${env:AMD_ENTRYPOINT}='vs/workbench/services/extensions/node/extensionHostProcess'; ${env:ANDROID_HOME}='C:\Dev\Android'; ${env:APPDATA}='C:\Users\woo\AppData\Roaming'; ${env:APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL}='true'; 
${env:CommonProgramFiles}='C:\Program Files\Common Files'; ${env:CommonProgramFiles(x86)}='C:\Program Files (x86)\Common Files'; ${env:CommonProgramW6432}='C:\Program Files\Common Files'; ${env:COMPUTERNAME}='DESKTOP-HU1TADK'; ${env:ComSpec}='C:\WINDOWS\system32\cmd.exe'; ${env:DriverData}='C:\Windows\System32\Drivers\DriverData'; ${env:ELECTRON_RUN_AS_NODE}='1'; ${env:FPS_BROWSER_APP_PROFILE_STRING}='Internet Explorer'; ${env:FPS_BROWSER_USER_PROFILE_STRING}='Default'; ${env:HOMEDRIVE}='C:'; 
${env:HOMEPATH}='\Users\woo'; ${env:IntelliJ IDEA}='C:\Dev\Java\IntelliJ IDEA 2019.1\bin;'; ${env:JAVA_HOME}='C:\Dev\Java\jdk1.8.0_201'; ${env:LOCALAPPDATA}='C:\Users\woo\AppData\Local'; ${env:LOGONSERVER}='\\DESKTOP-HU1TADK'; ${env:NUMBER_OF_PROCESSORS}='12'; ${env:OS}='Windows_NT'; ${env:PATHEXT}='.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW'; ${env:PIPE_LOGGING}='true'; ${env:PROCESSOR_ARCHITECTURE}='AMD64'; ${env:PROCESSOR_IDENTIFIER}='Intel64 Family 6 Model 158 Stepping 
10, GenuineIntel'; ${env:PROCESSOR_LEVEL}='6'; ${env:PROCESSOR_REVISION}='9e0a'; ${env:ProgramData}='C:\ProgramData'; ${env:ProgramFiles}='C:\Program Files'; ${env:ProgramFiles(x86)}='C:\Program Files (x86)'; ${env:ProgramW6432}='C:\Program Files'; ${env:PSModulePath}='C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules'; ${env:PUBLIC}='C:\Users\Public'; ${env:SESSIONNAME}='Console'; ${env:SystemDrive}='C:'; ${env:SystemRoot}='C:\WINDOWS'; ${env:TEMP}='C:\Users\woo\AppData\Local\Temp'; ${env:TMP}='C:\Users\woo\AppData\Local\Temp'; ${env:USERDOMAIN}='DESKTOP-HU1TADK'; ${env:USERDOMAIN_ROAMINGPROFILE}='DESKTOP-HU1TADK'; ${env:USERNAME}='woo'; ${env:USERPROFILE}='C:\Users\woo'; ${env:VBOX_MSI_INSTALL_PATH}='C:\Program Files\Oracle\VirtualBox\'; ${env:VERBOSE_LOGGING}='true'; ${env:VSCODE_CWD}='C:\Users\woo\AppData\Local\Programs\Microsoft VS Code'; ${env:VSCODE_HANDLES_UNCAUGHT_ERRORS}='true'; ${env:VSCODE_IPC_HOOK}='\\.\pipe\bf540a6f1434f810277c8e9e397d1e34-1.38.1-main-sock'; ${env:VSCODE_IPC_HOOK_EXTHOST}='\\.\pipe\vscode-ipc-f961af96-c2de-4ecd-be36-cdc2e92ab0ae-sock'; ${env:VSCODE_LOGS}='C:\Users\woo\AppData\Roaming\Code\logs\20190925T161650'; ${env:VSCODE_LOG_STACK}='false'; ${env:VSCODE_NLS_CONFIG}='{"locale":"ko","availableLanguages":{},"_languagePackSupport":true}'; ${env:VSCODE_NODE_CACHED_DATA_DIR}='C:\Users\woo\AppData\Roaming\Code\CachedData\b37e54c98e1a74ba89e03073e5a3761284e3ffb0'; ${env:VSCODE_PID}='11732'; ${env:VSCODE_PREVENT_FOREIGN_INSPECT}='true'; ${env:windir}='C:\WINDOWS'; ${env:Path}='C:\Dev\Python\Python37\Scripts\;C:\Dev\Python\Python37\;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files (x86)\dotnet\;C:\Program Files\PuTTY\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client 
SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Dev\Java\IntelliJ IDEA 2019.1\bin;C:\Dev\Java\jdk1.8.0_201\bin;C:\Program Files\Tesseract-OCR;C:\Users\woo\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files (x86)\Microsoft\ILMerge;C:\Users\woo\AppData\Local\Microsoft\WindowsApps;C:\Users\woo\AppData\Roaming\npm;C:\Program Files\nodejs;C:\Dev\Android\Flutter\bin;'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Dev\Python\Python37\python.exe' 'c:\Users\woo\.vscode\extensions\ms-python.python-2019.9.34911\pythonFiles\ptvsd_launcher.py' '--default' '--nodebug' '--client' '--host' 'localhost' '--port' '58517' 'c:\Dev\Python\workspace\lf\LFR

Steps to reproduce or exception report
--------------------------------------

마지막 200 키:
 e r s \ w o o \ . v s c o d e \ e x t e n s i o n s \ m s - p y t h o n . p y t h o n - 2 0 1 9 . 9 . 3 4 9 1 1 \ p y t h o n F i l e s \ p t v s d _ l a u n c h e r . p y ' Space ' - - d e f a u l t ' Space ' - - n o d e b u g ' Space ' - - c l i e n t ' Space ' - - h o s t ' Space ' l o c a l h o s t ' Space ' - - p o r t ' Space ' 5 8 5 1 7 ' Space ' c : \ D e v \ P y t h o n \ w o r k s p a c e \ l f \ L F R m s . p y ' Space Enter

예외:
System.ArgumentOutOfRangeException: 값은 0보다 크거나 같아야 하며 해당 범위에서 콘솔의 버퍼 크기보다 작아야 합니다.
매개 변수 이름: top
실제 값은 -4입니다.
   위치: System.Console.SetCursorPosition(Int32 left, Int32 top)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   위치: Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   위치: Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   위치: Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   위치: Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------

**Comment by nicholasmordecai** [comment]
I've just started getting this error too.. here's a log dump:

```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 } = ' K P A L O S I N D T R S D A F E ' ; Space $ { e n v : S E N D _ G R I D _ K E Y I D } = ' A A A A A A A a a a a A A A A ' ; Space & Space ' C : \ P r o g r a m Space F i 
l e s \ n o d e j s \ n o d e . e x e ' Space ' - - i n s p e c t - b r k = 3 7 3 7 2 ' Space ' d i s t \ i n d e x . j s ' Space Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -8.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```

**Comment by daxian-dbw** [comment]
> https://github.com/lzybkr/PSReadLine/issues/new

This URL indicates that you are using `2.0.0-beta3` or prior versions of PSReadLine. The issue should be fixed in the latest `2.0.0-beta5` version, please upgrade from PowerShellGallery.
